apps/dsaparam.c

```diff
@@ -266,11 +264,8 @@ int dsaparam_main(int argc, char **argv)
             ERR_print_errors(bio_err);
             goto end;
         }
-        if (!EVP_PKEY_keygen(ctx, &pkey)) {
-            BIO_printf(bio_err, "unable to generate key\n");
-            ERR_print_errors(bio_err);
+        if (!app_keygen(ctx, &pkey, verbose ? "DSA" : NULL, numbits))
             goto end;
```

Comment:
The point of being able to remove all the inner `ERR_print_errors()` is that
there is now a global one just before exit (also in the bail out case).

Reply:
I understood that that was the intent; I just failed to find the corresponding `ERR_print_errors()` call for this app.  Which is weird, because it seems like it's always been there.
Sorry for the noise.